### PR TITLE
Chore: JavaDoc fixes; dangling blocks, grammar, etc

### PR DESCRIPTION
**Pull request by dbmalkovsky**
There are lots of files but no code was changed (except for an occasional reordering of import statements).

Standardized the file headers (found lots of the same and duplicated [and wrong]) comments; removed or corrected parameter definitions, standardize capitalization of sentences (or fragments),  grammar changes, etc.   Just a lot of tedious editing.

Found with IntelliJ's Analyze plugin

### CODE REVIEW

File: src/main/java/org/isf/utils/db/DbQuery.java [hunk 1]
```diff
@@ -114,7 +109,7 @@ public ResultSet setDataReturnGeneratedKey(String aQuery,boolean autocommit) thr
      * @throws SQLException
      * @throws IOException
      * 
-     * @Deprecated since Open Hospital 1.7.0 replaced by {@link DbQueryLogger}
+     * @deprecated eprecated since Open Hospital 1.7.0 replaced by {@link DbQueryLogger}
```
Comment (mwithi) [inline src/main/java/org/isf/utils/db/DbQuery.java:112]: Missing "D" in deprecated

File: src/main/java/org/isf/utils/db/DbSingleConn.java [hunk 2]
```diff
@@ -37,15 +37,11 @@
...
 
-/*
- * @version 0.1 2005-11-06
- * @author bob
- *
- */
-
 /**
- * classe singleton che provvede alla connessione con database i parametri: dati
+ * Classe singleton che provvede alla connessione con database i parametri: dati
```
Comment (mwithi) [inline src/main/java/org/isf/utils/db/DbSingleConn.java:41]: Please change it in English
"Singleton class that provides connection with the database"

File: src/main/java/org/isf/utils/treetable/AbstractTreeTableModel.java [hunk 3]
```diff
@@ -60,21 +60,17 @@
...
 import javax.swing.event.TreeModelEvent;
 import javax.swing.event.TreeModelListener;
 import javax.swing.tree.TreePath;
  
 /**
  * An abstract implementation of the TreeTableModel interface, handling 
- * the list of listeners. 
- *
- * @version %I% %G%
+ * the list of listeners.
```
Comment (mwithi) [inline src/main/java/org/isf/utils/treetable/AbstractTreeTableModel.java:70]: Maybe it could be in one line only?

**Review by mwithi — CHANGES_REQUESTED** [review]
Small minor changes...

File: src/main/java/org/isf/utils/db/DbQuery.java (hunk 1 above)
Comment (dbmalkovsky) [reply src/main/java/org/isf/utils/db/DbQuery.java:112]: OK; done

File: src/main/java/org/isf/utils/db/DbSingleConn.java (hunk 2 above)
Comment (dbmalkovsky) [reply src/main/java/org/isf/utils/db/DbSingleConn.java:41]: Now in English

File: src/main/java/org/isf/utils/treetable/AbstractTreeTableModel.java (hunk 3 above)
Comment (dbmalkovsky) [reply src/main/java/org/isf/utils/treetable/AbstractTreeTableModel.java:70]: Removed all the version %x% strings in the files copied from Sun Microsystems.

**Review by mwithi — APPROVED** [review]

**Pull request merged**